mis_builder/models/simple_array.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# © 2014-2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+""" A trivial immutable array that supports basic arithmetic operations.
```

Comment:
@lepistone it is. I kind of thought pulling numpy was not appropriate :wink: 
In the end SimpleArray does a bit more wrt exception handling (DataError).